### ISSUE TITLE
Edit content field and remove type field from check.yml credits file

### ISSUE THREAD

**Issue by riddle015** [hackforla/website#2806]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/check.yml`, do the following:
- [x] Change
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove the following line:
  ```
  type: icon
  ```
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/check.yml`
- Click to see the source file code: https://github.com/hackforla/website/blob/gh-pages/_data/internal/credits/check.yml

**Comment by JessicaLucindaCheng** [comment]
Reopening this issue because the requested changes have not been implemented by a new developer.

**Comment by AsherEngelberg** [comment]
1. Progress: Have forked the repository and figured out how to actually open and edit the code in VS code. Finished the issue and resolved the redundancy.
2. Having trouble with how to push the committed changes to the gitHub repository to submit to the pull request.
3. I have a lot of time this week to finish the issue.
4. I expect that this issue will be completed within the next week as it is a small issue and I just need to familiarize myself more with the process.

**Comment by bishrfaisal** [comment]
@ExperimentsInHonesty The requested changes have not been implemented by the developer, as seen in the following screenshot.  
![Capture](https://user-images.githubusercontent.com/112529990/192836691-28b84682-f420-4343-bbae-82b914d60054.JPG)
Sending back to dev lead and adding to new issue approval column.

**Comment by JessicaLucindaCheng** [comment]
Reopening issue because the changes requested in the issue were not implemented yet.
